Makefile

```diff
@@ -1,5 +1,6 @@
 THIS_MAKEFILE = $(lastword $(MAKEFILE_LIST))
 REPOROOT = $(abspath $(dir $(THIS_MAKEFILE)))
+TEST_PACKAGES = $(subst $(REPOROOT)/src/,,$(shell go list -f '{{if gt (len .TestGoFiles) 0}}{{.Dir}}{{end}}' ./...))
```

Comment:
Do we want to exclude vendored packages from this?

Reply:
Go already excludes them be default. I believe it ignores the `vendor` dir by convention.